### ISSUE TITLE
Publish chart to a public helm repository

### ISSUE THREAD

**Issue by ahsan-storm** [upmc-enterprises/elasticsearch-operator#275]
Please get elastic-search operator & elasticsearch listed in the public helm repository i.e. https://github.com/helm/charts

**Comment by ahsan-storm** [comment]
@stevesloka

**Comment by pms1969** [comment]
or here https://hub.helm.sh/

**Comment by stevesloka** [comment]
Yup we can do that. Do you want to help out with a PR? I can get to it, will just take some time.
